<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Microchip Peripheral I/O Control</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <widget class="QPushButton" name="btn_exit">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabs">
      <property name="font">
       <font>
        <family>Liberation Sans</family>
       </font>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab_led">
       <attribute name="title">
        <string>LED</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_gpio">
       <attribute name="title">
        <string>GPIO</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_adc">
       <attribute name="title">
        <string>ADC</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_pwm">
       <attribute name="title">
        <string>PWM</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>About</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QPlainTextEdit" name="plainTextEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="plainText">
           <string>This application is written in Python, uses PyQt for the GUI, and uses the Microchip Peripheral I/O Python package to interface with the hardware.

The Microchip Peripheral I/O (mpio) package provides a Python API to many of the hardware peripherals found on boards running Linux like the Microchip SAMA5D2 Xplained, Microchip SAMA5D3 Xplained, and Microchip SAMA5D4 Xplained.

Microchip Peripheral I/O allows you to read and write to these hardware peripherals in an easy and consistent manner from Python.  Additionally, there are utilities that make working with these boards easier. The API is tested and there are plenty of common use case examples.

Supported interface classes include ADC, GPIO, I2C, SMBus, Input, LED, PWM, and SPI, Serial, and DevMem.</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
